student.rb

```diff
@@ -0,0 +1,21 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+  end
```

Comment:
- [ ] please like the previous reviewer noted. ensure that the Constructor extends parent's constructor by adding @classroom and a parameter for it.
example 

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```